tests/baselines/reference/declarationEmitNoInvalidCommentReuse2.js

```diff
@@ -0,0 +1,29 @@
+//// [tests/cases/compiler/declarationEmitNoInvalidCommentReuse2.ts] ////
+
+//// [a.ts]
+import { object } from "./obj.ts";
+
+export const _ = object;
+
+///////////
+/**
+ * huh
+ */
+//// [obj.d.ts]
+export declare const object: import("./id.ts").Id<{
+    foo: import("./id.ts" ).Id<{}>;
+}>;
+
+//// [id.d.ts]
+export type Id<T> = T;
+
+
+
+
+//// [a.d.ts]
+export declare const _: {
+    foo: import("./id").Id<{}>;

```

Comment:
I'm slightly surprised this one gets rewritten to `"./id"` (this test is using `rewriteRelativeImportExtensions`). IIRC I couldn't make it to be rewritten to anything else with any module-related options but maybe I have not tried hard enough. I guess it might behave differently from what `rewriteRelativeImportExtensions` promises us normally because it's in a type node.

Reply:
See: https://github.com/microsoft/TypeScript/issues/61037